cmd/cofidectl/cmd/trustzone/trustzone_test.go

```diff
@@ -53,6 +53,7 @@ func TestValidateOpts(t *testing.T) {
 		{domain: "valid.com", oidcIssuerURL: "https://validwithport.oidc:644", errExpected: false},
 		{domain: "valid.com", oidcIssuerURL: "h://invalid.oidc", errExpected: true},
 		{domain: "INVALID.COM", oidcIssuerURL: "https://valid.oidc", errExpected: true},
+		{domain: "INVALID.COM", oidcIssuerURL: "http://valid.oidc", errExpected: true},
```

Comment:
There are two reasons this could error so without checking the error message it might not be testing that http is not allowed.